CHANGELOG.md

```diff
@@ -60,10 +90,11 @@ and this project adheres to [Semantic Versioning].
 - Information about `PUSH0` instruction from [EIP-3855](https://eips.ethereum.org/EIPS/eip-3855)
   for Shanghai revision.
   [#628](https://github.com/ethereum/evmc/pull/628)
-- The [Paris](https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/paris.md)
-  (aka The Merge) EVM revision.
-  [#627](https://github.com/ethereum/evmc/pull/627)
-  [#634](https://github.com/ethereum/evmc/pull/634)
+-
```

Comment:
This looks to be accidental extra newline?

Reply:
I guess it's because of long URL line split by some auto-formatter

but it doesn't look good in rendered view:
![image](https://github.com/user-attachments/assets/54e16cdf-65f9-478a-a057-0ceca2b584f6)

Reply:
Woopsie. The formatting changes were not intended. Updated now with some additional typo fixes (there spell checks are getting better).